packages/node/src/handlers.ts

```diff
@@ -326,7 +327,7 @@ interface SentryTrpcMiddlewareOptions {
 
 interface TrpcMiddlewareArguments<T> {
   path: string;
-  type: 'query' | 'mutation' | 'subscription';
+  type: string;
```

Comment:
Are we broading this on purpose here (just checking)

Reply:
Yes, we are not depending on any of the direct strings here (just dumping what's in type onto the event) and I figured that if trpc adds more types we will break typing again.